### DISCUSSION TITLE
part-ii/2014-18/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#429] [Question comments]
Part II 2014 Asymptotic Methods: Paper 1, Section II, C 

https://questions.tripos.org/part-ii/2014-18/

**Comment by edwintang903** [comment]
(b) Change variables $t=ku$ to get leading order asymptotic approximation $\sqrt{\frac{k\pi}{3}}e^{-3k}$
